client/src/components/body/profilepage/EnrolledGroups.js

```diff
@@ -34,67 +18,86 @@ const EnrolledGroups = (props) => {
+    >
+      <div
+        className={"tw-h-full tw-flex tw-flex-col tw-justify-center tw-w-3/4"}
+      >
+        <div
+          className="tw-h-3/4 tw-bg-white tw-border-solid tw-rounded-bl-xl tw-border-primary-yellow
+                          tw-border-t-0 tw-border-r-0 tw-border-b-[1rem] tw-border-l-[1rem]"
+        >
           <div className="header_with_button">
-            <h4>My Enrolled Groups</h4>
-            <AddModal
-              addMode={"enroll_grp"}
-              user={props.user}
-              groupsUpdated={setGroupsUpdated}
-            />
+            <h4 className={"tw-title-styling-name tw-font-poppins tw-text-2xl"}>
+              {" "}
+              View Your Enrolled Groups{" "}
+            </h4>
           </div>
-          <div className="enrolled-groups">
-            {enrolledGroups.map((group, index) => (
-              <ul key={index}>
-                {index > 0 ? <hr className="groups__horiz" /> : <></>}
-                <ul className="groups" key={index}>
-                  <ul className="groups__group">
-                    <li className="groups__instructorName">
-                      <InstructorName instructorID={group.instructorUserID} />
-                    </li>
-                    <li className="groups__groupName">{group.groupName}</li>
-                  </ul>
-                  <ul className="groups__group">
-                    <GroupDetails group={group} instructing={false} />
-                  </ul>
-                  <ul className="groups__group">
-                    <li className="groups__date">
-                      Enrolled on {group.enrolledDate.split("T")[0]}
-                    </li>
-                    <li>
-                      <UnenrollModal
-                        userid={user.userid}
-                        groupid={group.groupID}
-                        buttonLabel={"Unenroll"}
-                        groupsUpdated={setGroupsUpdated}
-                      />
-                    </li>
-                  </ul>
-                </ul>
-              </ul>
-            ))}
+          <div
+            className={"tw-flex tw-flex-row tw-justify-between tw-h-3/4 tw-m-5"}
+          >
+            {enrolledGroups.length === 0 ? (
+              <div>
+                <p> You are currently not enrolled in any groups</p>
+              </div>
+            ) : (
+              <div
+                className={
+                  "xs:tw-flex xs:tw-flex-col md:lg:tw-grid md:lg:tw-grid-cols-3 tw-gap-3"
+                }
+              >
+                {displayEnrolledGroups()}
+              </div>
+            )}
+            <div className={"tw-flex tw-flex-col tw-text-left"}>
+              <p className={"tw-text-lg tw-title-styling-name tw-font-poppins"}>
+                {" "}
+                Have a group code?{" "}
+              </p>
+              <p className={"tw-text-sm tw-font-calibri"}>
+                {" "}
+                Click below to get started.{" "}
```

Comment:
The font sizes in the tailwind config file all need to get updated. It says this is 14px, should be 18px. Fix the tailwind file to have the right font sizes